File: c++works/Classes/Classes/Circle.cpp
//#include <iostream>
//using namespace std;
//
////Circle 클래스 정의
//class Circle{
//public:
//	int radius;
//
//	//생성자
//	Circle();
//	Circle(int r);
//
//	double getArea();
//};
//
//Circle::Circle() 
//{
//	radius = 1;
//	cout << "반지름" << radius << "인 원 생성" << endl;
//}
//
//Circle::Circle(int r)
//{
//	radius = r;
//	cout << "반지름" << radius << "인 원 생성" << endl;
//}
//
//double Circle::getArea() 
//{
//	return 3.14 * radius * radius;
//}
//
//int main()
//{
//	//반지름이 1인 원 생성
//	Circle c1;
//	cout << "원의 면적은 " << c1.getArea() << endl;
//
//	cout << "==========================\n";
//
//	//반지름이 10인 원 생성
//	Circle c2(10);
//	cout << "원의 면적은 " << c2.getArea() << endl;
//
//	return 0;
//}

File: javaworks/Chapter6/src/reference/Circle.java
package reference;

public class Circle {
	Point center;  //중심점
	int radius;    //반지름
	
	public Circle(int x, int y, int radius) {
		center = new Point(x, y);
		this.radius = radius;
	}
	
	public void showInfo() {
		System.out.println("원의 중심은 (" + center.x + ", " + center.y + 
				")이고, 반지름은 " + radius + "입니다.");
	}

}
